packages/global/scss/core.scss

```diff
@@ -279,7 +279,7 @@
         // stylelint-disable-next-line selector-max-compound-selectors
         &__section {
           // stylelint-disable-next-line selector-max-compound-selectors
-          > .social-icon-link {
+          .social-icon-link {
```

Comment:
Was this intended to be removed? Not seeing anything changed within it to warrant this.

Reply:
@Shinsina I thought it was added accidentally?